dcrwallet.go

```diff
@@ -145,6 +145,11 @@ func walletMain() error {
 		if !cfg.NoInitialLoad {
 			passphrase = startPromptPass(w)
 		}
+
+		// Set the password if provided in the config
+		if len(cfg.Pass) > 0 {
+			passphrase = []byte(cfg.Pass)
```

Comment:
Couple things need to be fixed here:

First, move this check up into the earlier if block.  It should read something like:

```
if !cfg.NoInitialLoad {
    if cfg.Pass != "" {
        passphrase = []byte(cfg.Pass)
    } else {
        passphrase = startPromptPass(w)
    }
}
```

Without this change it will just overwrite the passphrase it just prompted for.

Secondly it needs to actually unlock the wallet using this passphrase (and verify that the passphrase was correct).  Since the wallet is opened at this point, we can perform this check.  startPromptPass already does this by calling w.SetInitiallyUnlocked(true) and then unlocking the wallet with w.Unlock.  If this unlock step errors, then the error should be returned.